packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -30,6 +30,8 @@ export interface BaseRichTextEditorProps
    * The editable area will autogrow, so this only affects the component when the content doesn't exceed this height.
    */
   rows?: EditorRows
+  dataError?: React.ReactElement
```

Comment:
Would ReactElement be best here? Or ReactNode? Do we want consumers to pass in raw strings, or should we restrict it to elements only?